pyproject.toml

```
@@ -5,7 +5,7 @@ build-backend = "flit_core.buildapi"
 [project]
 # Check https://flit.readthedocs.io/en/latest/pyproject_toml.html for all available sections
 name = "pyconverter-xml2py"
-version = "0.1.dev0"
+version = "0.2.dev0"
```

```suggestion
version = "0.3.dev0"
```

Should be 0.3.dev0 since 0.2.0 has already been released. Main typically stays on the next minor version

---

Got it, thank you!